### PR TITLE
ec2 instance

### PR DESCRIPTION
**Pull request by vikrant-subhash**

### CODE REVIEW

File: instances.tf [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  tags = {
+    Name = "${var.env_code}-public"
+  }
+
+}
+
+# Public Security groups
+resource "aws_security_group" "Public_SG" {
+  name        = "TF_PubSG"
```
Comment (vladimirmukhin) [inline instances.tf:17]:
```suggestion
  name        = "${var.env_code}-public"
```

File: instances.tf [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  egress {
+    from_port        = 0
+    to_port          = 0
+    protocol         = "-1"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
+  }
+
+  tags = {
+    Name = "Public_SG"
```
Comment (vladimirmukhin) [inline instances.tf:38]:
```suggestion
    Name = "${var.env_code}-public"
```

File: instances.tf [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  instance_type          = "t3.micro"
+  key_name               = "terraform_key_pair"
+  vpc_security_group_ids = [aws_security_group.Private_SG.id]
+  subnet_id              = aws_subnet.private[0].id
+
+  tags = {
+    Name = "${var.env_code}-private"
+  }
+
+}
```
Comment (vladimirmukhin) [inline instances.tf:53]:
```suggestion
  }
}
```

File: instances.tf [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  egress {
+    from_port        = 0
+    to_port          = 0
+    protocol         = "-1"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
+  }
+
+  tags = {
+    Name = "Private_SG"
```
Comment (vladimirmukhin) [inline instances.tf:79]:
```suggestion
    Name = "${var.env_code}-private"
```

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]
"Private_SG" and "Public_SG" aren't good names for your security groups. Please rename them to "private" and "public"

**Comment by vikrant-subhash** [comment]
creating new pull request

**Pull request closed without merging**